### DISCUSSION TITLE
What common word has ...

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#640] [Galaxy Trivia] [answered]
... 4 vowels, one after the other?

**Comment by DevElkami** [comment, marked as answer]
Queue is the answer.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/DevElkami?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Comment by peterbaccaro** [comment]
QUEUE

![queue](https://user-images.githubusercontent.com/74666079/235161161-eb12300d-c1fa-4458-8afd-73e0e69dd4de.jpg)
